### PR TITLE
[RM-5211] Google Environment support

### PR DESCRIPTION
**Pull request by ricleal-fugue**
Support for creating an environment, update it and get it.

### CODE REVIEW

File: cmd/getEnvironment.go [hunk 1]
```diff
@@ -99,6 +99,8 @@ func NewGetEnvironmentCommand() *cobra.Command {
 			case "azure":
 				items = append(items, Item{"SUBSCRIPTION_ID", env.ProviderOptions.Azure.SubscriptionID})
 				items = append(items, Item{"APPLICATION_ID", env.ProviderOptions.Azure.ApplicationID})
+			case "google":
+				items = append(items, Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID})
```
Comment (mattm-fugue) [inline cmd/getEnvironment.go:103]: Is the Service Account Email also returned here?

File: cmd/createGoogleEnvironment.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+			items := []interface{}{
+				Item{"ENVIRONMENT_ID", env.ID},
+				Item{"NAME", env.Name},
+				Item{"PROVIDER", env.Provider},
+				Item{"SCAN_INTERVAL", env.ScanInterval},
+				Item{"LAST_SCAN_AT", format.Unix(env.LastScanAt)},
+				Item{"NEXT_SCAN_AT", format.Unix(env.NextScanAt)},
+				Item{"SCAN_STATUS", env.ScanStatus},
+				Item{"COMPLIANCE_FAMILIES", families},
+				Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID},
```
Comment (mattm-fugue) [inline cmd/createGoogleEnvironment.go:73]: The AWS / Azure commands don't return any of the environment specific metadata in this table. This seems like a good change - should we propagate the concept out to those create/update commands?

File: cmd/getEnvironment.go (hunk 1 above)
Comment (mattm-fugue) [inline cmd/getEnvironment.go:103]: Should this display Service Account Email as well?

Comment (ricleal-fugue) [reply cmd/getEnvironment.go:103]: I can print it out but I see the email as "confidential"...

File: cmd/createGoogleEnvironment.go (hunk 2 above)
Comment (ricleal-fugue) [reply cmd/createGoogleEnvironment.go:73]: This is the create... What do you mean by propagating this to "create/update commands"?
The update has already a:
```
			case "google":
				items = append(items, Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID})
			}
```

**Pull request merged**